modules/stanford_profile_styles/lib/scss/component/paragraph/card/_card.scss

```diff
@@ -14,6 +17,30 @@
       @include type-e;
     }
 
+    h2,
+    h3,
+    h4 {
+      flex-shrink: 0;
+      order: 2;
+
+      &:first-child {
+        margin-top: 0;
+      }
+    }
+
+    .su-card__superhead {
+      order: 1;
+      margin-top: -0.3em;
+    }
+
+    p {
+      order: 3;
+    }
```

Comment:
Not all contents in the WYSIWYG will be a `<p>`.

My recommendation is to only set the `order` on the heading and the super head. Leave the rest to flow naturally. They default to `order: 0` and then will display in the order of their dom. If you set `order: -1` only to the superhead, it will pull itself up. Similarly to how tailwind does it: `order-first	order: -9999;`: https://v3.tailwindcss.com/docs/order

<img width="668" height="518" alt="image" src="https://github.com/user-attachments/assets/ba0af404-a9ee-4127-ad67-7756d0022fa4" />

Reply:
Ah! I was just thinking this morning about how Tailwind's ordering works and that makes so much more sense 🤦 

Thank you Mike! I'll get this fixed up.